shared/data_lake.py

```diff
@@ -1,29 +1,28 @@
-import logging
-import os
-
-from azure.storage.filedatalake import DataLakeServiceClient
-
-
-# RBAC role - Storage Blob Data Owner
-def upload_to_data_lake(azure_credential, file_name, data_str):
-
-    account_name = os.environ.get("DATALAKE_GEN_2_RESOURCE_NAME")
-    container_name = os.environ.get("DATALAKE_GEN_2_CONTAINER_NAME")
-    directory_name = os.environ.get("DATALAKE_GEN_2_DIRECTORY_NAME")
-
-    unique_file_path = f"{file_name}"

```

Comment:
unique_file_path seems to be exact same as file_name so not sure why new variable is needed?